### DISCUSSION TITLE
Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project cache: There are test failures.

### DISCUSSION THREAD

**Discussion by nickatnite4** [runelite/runelite#16951] [Q&A]
I working through Runelite's wiki on Building with [IntelliJ IDEA](https://github.com/runelite/runelite/wiki/Building-with-IntelliJ-IDEA#troubleshooting) and I can't seem to build the project correctly I keep running into the following error

### Error:
`Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.2:test (default-test) on project cache: There are test failures.`

### Images of Error:

![error-1](https://github.com/runelite/runelite/assets/20198316/a453bdc8-e074-445f-9aa3-b22de2fcc97e)

![error-2](https://github.com/runelite/runelite/assets/20198316/6cef3879-c288-4cc6-ac16-e97ce01289cf)

Steps Followed:

![runelite – README md  runelite-parent  8_22_2023 6_27_25 AM (2)](https://github.com/runelite/runelite/assets/20198316/41e96531-cc9b-4d6f-a9b9-87681e391d11)

Then i Click `Run Maven Build`

This is when i get the error.

I am using Java 11.

![runelite – README md  runelite-parent  8_22_2023 6_31_07 AM](https://github.com/runelite/runelite/assets/20198316/84aad48a-9cd3-407d-bb21-4dbcfa3f67b2)

Can't seem to get past this build issue for the past few days.

**Comment by raiyni** [comment]
Add `-DskipTests` to your maven run options. If you need further build help, you can ask in the #runelite discord channel.
